packages/blockchain-ui/package-lock.json

```diff
@@ -14,9 +14,9 @@
 			}
 		},
 		"@babel/compat-data": {
-			"version": "7.8.5",
-			"resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.8.5.tgz",
-			"integrity": "sha512-jWYUqQX/ObOhG1UiEkbH5SANsE/8oKXiQWjj7p7xgj9Zmnt//aUvyz4dBkK0HNsS8/cbyC5NmmH87VekW+mXFg==",
+			"version": "7.8.6",
```

Comment:
you don't need these changes you haven't changed any version in the package.json file